build.py

```diff
@@ -331,45 +253,67 @@ def apply_patches():
         i = re.sub("TEMPLATE_AVRO_RPATH", avro_rpath, i)
         i = re.sub("TEMPLATE_AVRO_PATH", avro_root, i)
         i = re.sub("TEMPLATE_CPPZMQ_PATH", cppzmq_root, i)
-        i = re.sub("TEMPLATE_FMT_PATH", fmt_root, i)
-        i = re.sub("TEMPLATE_FMT_RPATH", fmt_rpath, i)
-        i = re.sub("TEMPLATE_JSON_PATH", json_root, i)
-        run_cmd(i, run_env=myenv, unsafe_shell=True, check_rc='build failed')
+        run_cmd(i, unsafe_shell=True, check_rc='build failed')
 
     # package
     if not build_native_package:
         return
 
-    fpmbinary = 'fpm'
-    fpmbinary = which(fpmbinary)
-    if fpmbinary is None:
-        log.error('fpm not found, try "gem install fpm"')
-        sys.exit(1)
+    import yaml
+
     os.chdir(script_path)
-    package_cmd = [fpmbinary, '-f', '-s', 'dir']
-    package_cmd.extend(['-t', distro_info.package_type()])
-    package_cmd.extend(['-n', 'irods-externals-{0}'.format(package_subdirectory)])
-    for d in get_package_dependencies(v):
-        package_cmd.extend(['-d', d])
-    package_cmd.extend(['-m', '<packages@irods.org>'])
-    package_cmd.extend(['--version', get_package_version(target)])
-    package_cmd.extend(['--iteration', get_package_revision(target)])
-    package_cmd.extend(['--vendor', 'iRODS Consortium'])
-    package_cmd.extend(['--license', v['license']])
-    package_cmd.extend(['--description', 'iRODS Build Dependency'])
-    package_cmd.extend(['--url', 'https://irods.org'])
-    package_cmd.extend(['-C', build_dir])
+
+    nfpmbinary = 'nfpm'
+    nfpmbinary = which(nfpmbinary)
+    if nfpmbinary is None:
+        log.error('nfpm not found, see https://nfpm.goreleaser.com/install/')
+        sys.exit(1)
+
+    nfpm_cfg_dict = {
```

Comment:
Is this config tied to a specific version of nfpm?

Reply:
v2.40 or newer. License tag for deb packages was added then.